.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDateTimePeriodParserConfiguration.cs

```diff
@@ -148,10 +148,17 @@ public EnglishDateTimePeriodParserConfiguration(ICommonDateTimeParserConfigurati
         public bool GetMatchedTimeRange(string text, out string todSymbol, out int beginHour, out int endHour, out int endMin)
         {
             var trimmedText = text.Trim();
-
             beginHour = 0;
             endHour = 0;
             endMin = 0;
+
+            // Change for ToDo to match the new rules set by the NLP for DateTime project
```

Comment:
What does this mean? "by the NLP for DateTime project"

Reply:
I will change it to TasksMode